### PR TITLE
document driver config and defaults

### PR DESCRIPTION
**Pull request by dougwig**

### CODE REVIEW

File: a10_neutron_lbaas/etc/defaults.py [hunk 1]
```diff
@@ -41,10 +31,13 @@
     "port": 443,
     "api_version": "2.1",
     "status": True,
-    "autosnat": True,
+    "autosnat": False,
```
Comment (Cedev) [inline a10_neutron_lbaas/etc/defaults.py]: Why `'autosnat': False`? It's the acos default, but it's broken by default.

Comment (Cedev) [inline a10_neutron_lbaas/etc/defaults.py]: Why `'autosnat': False`? It's the acos default, but it's broken by default.

**Comment by Cedev** [comment]
Even though `'autosnat': True` should be the default, it's not the existing default for existing installations, so it should ship as `'autosnat': False`.

**Comment by Cedev** [comment]
+1 for documentation. -1. Tests are failing with a really weird error

```
2016-03-22 18:10:29.903 22415 ERROR neutron_lbaas.services.loadbalancer.plugin Traceback (most recent call last):
2016-03-22 18:10:29.903 22415 ERROR neutron_lbaas.services.loadbalancer.plugin   File "/opt/stack/new/neutron-lbaas/neutron_lbaas/services/loadbalancer/plugin.py", line 486, in _call_driver_operation
2016-03-22 18:10:29.903 22415 ERROR neutron_lbaas.services.loadbalancer.plugin     driver_method(context, db_entity)
2016-03-22 18:10:29.903 22415 ERROR neutron_lbaas.services.loadbalancer.plugin   File "/opt/stack/new/neutron-lbaas/neutron_lbaas/drivers/a10networks/driver_v2.py", line 44, in create
2016-03-22 18:10:29.903 22415 ERROR neutron_lbaas.services.loadbalancer.plugin     self.driver.a10.lb.create(context, lb)
2016-03-22 18:10:29.903 22415 ERROR neutron_lbaas.services.loadbalancer.plugin   File "/usr/local/lib/python2.7/dist-packages/a10_neutron_lbaas/v2/handler_lb.py", line 49, in create
2016-03-22 18:10:29.903 22415 ERROR neutron_lbaas.services.loadbalancer.plugin     with a10.A10WriteStatusContext(self, context, lb) as c:
2016-03-22 18:10:29.903 22415 ERROR neutron_lbaas.services.loadbalancer.plugin   File "/usr/local/lib/python2.7/dist-packages/a10_neutron_lbaas/a10_context.py", line 43, in __enter__
2016-03-22 18:10:29.903 22415 ERROR neutron_lbaas.services.loadbalancer.plugin     self.select_appliance_partition()
2016-03-22 18:10:29.903 22415 ERROR neutron_lbaas.services.loadbalancer.plugin   File "/usr/local/lib/python2.7/dist-packages/a10_neutron_lbaas/a10_context.py", line 74, in select_appliance_partition
2016-03-22 18:10:29.903 22415 ERROR neutron_lbaas.services.loadbalancer.plugin     self.client.system.partition.active(name)
2016-03-22 18:10:29.903 22415 ERROR neutron_lbaas.services.loadbalancer.plugin   File "/usr/local/lib/python2.7/dist-packages/acos_client/v21/partition.py", line 33, in active
2016-03-22 18:10:29.903 22415 ERROR neutron_lbaas.services.loadbalancer.plugin     self._post("system.partition.active", {'name': name})
2016-03-22 18:10:29.903 22415 ERROR neutron_lbaas.services.loadbalancer.plugin   File "/usr/local/lib/python2.7/dist-packages/acos_client/v21/base.py", line 62, in _post
2016-03-22 18:10:29.903 22415 ERROR neutron_lbaas.services.loadbalancer.plugin     return self._request('POST', action, params, **kwargs)
2016-03-22 18:10:29.903 22415 ERROR neutron_lbaas.services.loadbalancer.plugin   File "/usr/local/lib/python2.7/dist-packages/acos_client/v21/base.py", line 37, in _request
2016-03-22 18:10:29.903 22415 ERROR neutron_lbaas.services.loadbalancer.plugin     return self.client.http.request(method, self.url(action), params,
2016-03-22 18:10:29.903 22415 ERROR neutron_lbaas.services.loadbalancer.plugin   File "/usr/local/lib/python2.7/dist-packages/acos_client/v21/base.py", line 30, in url
2016-03-22 18:10:29.903 22415 ERROR neutron_lbaas.services.loadbalancer.plugin     (action, self.client.session.id))
2016-03-22 18:10:29.903 22415 ERROR neutron_lbaas.services.loadbalancer.plugin   File "/usr/local/lib/python2.7/dist-packages/acos_client/v21/session.py", line 30, in id
2016-03-22 18:10:29.903 22415 ERROR neutron_lbaas.services.loadbalancer.plugin     self.authenticate(self.username, self.password)
2016-03-22 18:10:29.903 22415 ERROR neutron_lbaas.services.loadbalancer.plugin   File "/usr/local/lib/python2.7/dist-packages/acos_client/v21/session.py", line 43, in authenticate
2016-03-22 18:10:29.903 22415 ERROR neutron_lbaas.services.loadbalancer.plugin     r = self.http.post(url, params)
2016-03-22 18:10:29.903 22415 ERROR neutron_lbaas.services.loadbalancer.plugin   File "/usr/local/lib/python2.7/dist-packages/acos_client/v21/axapi_http.py", line 234, in post
2016-03-22 18:10:29.903 22415 ERROR neutron_lbaas.services.loadbalancer.plugin     return self.request("POST", api_url, params, **kwargs)
2016-03-22 18:10:29.903 22415 ERROR neutron_lbaas.services.loadbalancer.plugin   File "/usr/local/lib/python2.7/dist-packages/acos_client/v21/axapi_http.py", line 185, in request
2016-03-22 18:10:29.903 22415 ERROR neutron_lbaas.services.loadbalancer.plugin     raise e
2016-03-22 18:10:29.903 22415 ERROR neutron_lbaas.services.loadbalancer.plugin gaierror: [Errno -5] No address associated with hostname
```

**Comment by dougwig** [comment]
check

**Comment by dougwig** [comment]
check

**Pull request merged**